### DISCUSSION TITLE
Wave height pattern around hurricane

### DISCUSSION THREAD

**Discussion by Liu-Jincan** [NOAA-EMC/WW3#1075] [Q&A]
Hello everyone, I would like to ask if the wave height pattern around the hurricane shown in the middle part of the graph below is normal. I feel that this part is not very smooth and has many protrusions.  Any suggestions are welcome.
![mmexport1694844984514.jpg](https://github.com/NOAA-EMC/WW3/assets/73934823/bbfd4bde-6903-45f3-a406-853e26069617)

**Comment by aronroland** [comment]
hi liu, the pattern u can see shows the GSE effect. There are methods in WW3 to reduce this effect. There papers and this presentation at NCEP https://polar.ncep.noaa.gov/waves/workshop/pdfs/wwws_2013_GSE.pdf. I hope this helps.

**Reply by Liu-Jincan** [reply]
@aronroland Thank you very much for your valuable reply.  It is indeed the GSE phenomenon caused by the use of ultra-high resolution.  Prepare to refer to the relevant source item settings in the PDF for simulation.  Thanks again.

**Reply by aronroland** [reply]
u r welcome
